modules/OCPP/OCPP.cpp

```diff
@@ -355,6 +358,53 @@ void OCPP::init_evse_maps() {
     }
 }
 
+void OCPP::init_module_configuration() {
+    std::vector<ocpp::CiString<50UL>> keys;
```

Comment:
```suggestion
    std::vector<ocpp::CiString<50>> keys;
```

or is that needed in some way?